nest/command_line.py

```diff
@@ -227,6 +279,11 @@ def _identity(x):
             pin = input("PIN: ")
             napi.request_token(pin)
 
+        if not cmd:
```

Comment:
If this is not directly related to the streaming API, please submit changes into a pull request in the main repo.